android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -413,22 +410,34 @@ public void getAlbums(final ReadableMap params, final Promise promise) {
         WritableArray response = new WritableNativeArray();
         try {
           if (media.moveToFirst()) {
+            Map<String, Integer> albums = new TreeMap<>();
```

Comment:
What is the purpose of `TreeMap` here? If you just want to order album name, I think we do not need to do that. Basically, lib just return the album list, user may want to order the list by themself (by name or by count).

Reply:
that was mainly for ordering, I can change to `HashMap` if we don't care about order and eventually clients can order on their and if they need to. Should I?

Reply:
It would be great!